DemoApp/Sources/Views/CallView/DemoCallView.swift

```diff
@@ -76,6 +77,20 @@ struct DemoCallView<ViewFactory: DemoAppViewFactory>: View {
                 updateMicrophoneChecker()
             }
             .chat(viewModel: viewModel, chatViewModel: chatViewModel)
+            .snapshot(captureTrigger: snapshotTrigger.$capture.eraseToAnyPublisher()) { [weak viewModel, weak snapshotTrigger] in
+                guard let data = $0.pngData() else { return }
```

Comment:
maybe we could use jpeg with 0.3 compression and add a comment about the size limitation.

Reply:
I'm not sure i agree. Even though what you are saying may be valid for a specific usecase it may fail for others (as it will return a low-resolution image). IMO it makes sense return something that has full data and customers can lower the resolution as they need during their processing.

Reply:
The concern I have is that this request will almost never work - snapshot images would usually be over the limit. Another option I can think of is to actually not display it as part of the demo app.

Reply:
I think you have in your mind only the scenario where someone wants to send the snapshot via a custom-event. There are other scenarios like sharing it to chat or other applications where reducing quality will affect them. 

> Another option I can think of is to actually not display it as part of the demo app.
I'm not sure i understand this part. What option are you referring to? Currently we aren't displaying this image anywhere. We only did this part of the as an example.

Reply:
Well the sample code here sends custom events, and as far as I can see, the button to trigger this flow is visible. So what will happen is that whenever someone presses that button, the "limit" error will be thrown, right? If yes, then what's the point of having it visible or not having a size reduction?

Reply:
I just got what you were saying. This isn't a change on the SDK but on the DemoApp (the caller). It definitely makes sense to reduce it.